conversation/java/sdk/conversation/src/main/java/com/service/ConversationAIApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.service;
+
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.DaprPreviewClient;
+import io.dapr.client.domain.ConversationInput;
+import io.dapr.client.domain.ConversationRequest;
+import io.dapr.client.domain.ConversationResponse;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+public class ConversationAIApplication {
+
+    public static void main(String[] args) {
+        try (DaprPreviewClient client = new DaprClientBuilder().buildPreviewClient()) {
+            System.out.println("Sending the following input to LLM: Hello How are you? This is the my number 672-123-4567");
```

Comment:
As boring as it is, can we standardize on this: "What is Dapr?" 
https://github.com/dapr/quickstarts/blob/master/conversation/csharp/sdk/conversation/Program.cs#L38

And then make it a constant var